front/src/modules/ui/feedback/dialog/components/DialogManager.tsx

```diff
@@ -1,30 +1,18 @@
 import { useEffect } from 'react';
```

Comment:
I feel the whole folder should be renamed to DialogManager and we should only have useDialogManager (not useDialog)